cranelift/codegen/src/ir/entities.rs

```diff
@@ -88,12 +88,15 @@ impl Value {
 ///
 /// Most usage of `Inst` is internal. `Inst`ructions are returned by
 /// [`InstBuilder`](super::InstBuilder) instructions that do not return a
-/// [`Value`], such as control flow and trap instructions.
+/// [`Value`], such as control flow and trap instructions, as well as instructions that return a
+/// variable (potentially zero!) number of values, like call or call-indirect instructions.
```

Comment:
Maybe we can put a note here (perhaps instead of below) to the effect of: "... like call or call-indirect instructions. To get the `Value` results of such an instruction, use `inst_results` with the returned `Inst`" ?

Reply:
admittedly took me a moment to push the change but i really appreciate this suggestion! it cleans up the whole "talk about the normal thing, then the exotic thing, then the normal thing again" wrinkle nicely.